addons/mail/static/src/core/web/follower_subtype_dialog.js

```diff
@@ -28,13 +21,17 @@ export class FollowerSubtypeDialog extends Component {
         super.setup();
         this.store = useService("mail.store");
         this.state = useState({
-            /** @type {SubtypeData[]} */
+            /** @type {import("models").MailMessageSubtype[]} */
             subtypes: [],
         });
         onWillStart(async () => {
-            this.state.subtypes = await rpc("/mail/read_subscription_data", {
+            const { store_data, subtype_ids } = await rpc("/mail/read_subscription_data", {
                 follower_id: this.props.follower.id,
             });
+            this.store.insert(store_data);
+            this.state.subtypes = subtype_ids.map((id) =>
```

Comment:
Why do we keep a local state?
It's specially confusing if the template displays the checkbox state based on the value on the follower instead.